client/src/shared/components/Modal/Modal.tsx

```diff
@@ -22,9 +20,10 @@ const Modal: React.FC<ModalProps> = ({
           ></span>
 
           <div
-            className={`inline-block bg-white rounded-lg text-left overflow-hidden shadow-xl transform transition-all align-middle w-2/4 ${
-              className ?? ''
-            }`}
+            className={`inline-block bg-white rounded-lg text-left overflow-hidden shadow-xl transform transition-all align-middle ${

```

Comment:
```suggestion
          className={`inline-block bg-white rounded-lg text-left overflow-hidden shadow-xl transform transition-all align-middle w-2/4 ${
              className ?? ''
            }`}

```

since we can pass className as props, we dont need to pass any other props such as `width`, you can just add a className in the modal to overwrite the default width like this  `<Modal isOpen={isOpen} className='w-[600px]'>`

Reply:
Ohhh okay so we are overriding the fixed width. Thank you for this ster :bow: